modules/core/ui/views/config/optional/views.view.farm_organization.yml

```diff
@@ -0,0 +1,516 @@
+              authenticated: authenticated
+            reduce: false
+          is_grouped: false
+          group_info:
+            label: ''
+            description: ''
+            identifier: ''
+            optional: true
+            widget: select
+            multiple: false
+            remember: false
+            default_group: All
+            default_group_multiple: {  }
+            group_items: {  }
+      style:
+        type: table
+        options:
+          grouping: {  }
+          row_class: ''
+          default_row_class: true
+          columns:
+            organization_bulk_form: organization_bulk_form
+            name: name
+            type: type
+            status: status
+          default: '-1'
+          info:
+            organization_bulk_form:
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+            name:
+              sortable: true
+              default_sort_order: asc
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+            type:
+              sortable: true
+              default_sort_order: asc
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+            status:
+              sortable: true
+              default_sort_order: asc
+              align: ''
+              separator: ''
+              empty_column: false
+              responsive: ''
+          override: true
+          sticky: false
+          summary: ''
+          empty_table: false
+          caption: ''
+          description: ''
+      row:
+        type: fields
+      query:
+        type: views_query
+        options:
+          query_comment: ''
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_tags: {  }
+      relationships: {  }
+      header: {  }
+      footer:
+        result:
+          id: result
+          table: views
+          field: result
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: result
+          empty: false
+          content: 'Displaying @start - @end of @total'
+      display_extenders: {  }
+    cache_metadata:
+      max-age: 0
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - user.permissions
+      tags: {  }
+  page_1:
+    id: page_1
+    display_title: Page
+    display_plugin: page
+    position: 1
+    display_options:
+      display_extenders:
+        collapsible_filter:
+          collapsible: false
+      path: organizations
+      menu:
+        type: normal
+        title: Organizations
+        description: ''
+        weight: 0
+        expanded: true
+        menu_name: admin
+        parent: farm.setup
+        context: '0'
```

Comment:
Ah, having this menu link in the view means we don't need to worry about the dependency on organization module.. a little simpler in that regard.

I'll defer to you @mstenta on the description text

Reply:
Ah good point @paul121 - I didn't look at that page. I committed this as a first step:

> Manage the organizations that records can be associated with.

Might be able to improve it further. I started with a verb to maintain consistency with the other descriptions, but otherwise kept it simple. Open to suggestions (but remember that at this level it has to be agnostic to specific organization types, like `farm`).